src/standardMethods.ts

```diff
@@ -225,38 +229,52 @@ export function generateRetrieveSingleEndpoint<IdType, ResponseType>(
 export function generateDeleteEndpoint<IdType>(
   axiosInstance: AxiosInstance,
   resourcePath: string,
-  metadataMap: MetadataMap
+  metadataMap: MetadataMap,
+  chunkFunction?: (ids: IdType[]) => IdType[][]
 ) {
   return async function remove(ids: IdType[]): Promise<{}> {
-    const response = await rawRequest<{}>(
-      axiosInstance,
-      {
-        url: `${resourcePath}/delete`,
-        method: 'post',
-        data: { items: ids },
-      },
+    const chunks = doChunking<IdType>(ids, chunkFunction);
+    const responses = await promiseAllWithData(
+      chunks,
+      input =>
+        rawRequest<ItemsResponse<ResponseType>>(axiosInstance, {
+          method: 'post',
+          url: `${resourcePath}/delete`,
+          data: { items: input },
+        }),
       true
     );
-    return metadataMap.addAndReturn({}, response);
+    return metadataMap.addAndReturn({}, responses[0]);
   };
 }
 
 /** @hidden */
 export function generateUpdateEndpoint<RequestType, ResponseType>(
   axiosInstance: AxiosInstance,
   resourcePath: string,
-  metadataMap: MetadataMap
+  metadataMap: MetadataMap,
+  chunkFunction?: (changes: RequestType[]) => RequestType[][]
 ) {
   return async function update(
     changes: RequestType[]
   ): Promise<ResponseType[]> {
     type Response = ItemsResponse<ResponseType>;
-    const response = await rawRequest<Response>(axiosInstance, {
-      url: `${resourcePath}/update`,
-      method: 'post',
-      data: { items: changes },
-    });
-    return metadataMap.addAndReturn(response.data.items, response);
+    const chunks = doChunking<RequestType>(changes, chunkFunction);
+    const responses = await promiseAllWithData(
+      chunks,
+      input =>
+        rawRequest<Response>(axiosInstance, {
+          method: 'post',
+          url: `${resourcePath}/update`,
+          data: { items: input },
+        }),
+      true
```

Comment:
do we need an order here? I think we can change this to `false`

Reply:
Hm. We prob. want to keep the order as the same as you requested?

Reply:
the responses will be in the same order
this only says about the request order

Reply:
Oh wait, yes, I am wrong

Reply:
It does change the order 
assetUtils.ts:59: `responses.push(result);`

Reply:
But that probably means that we need to have separate pr to fix that
Because sending requests sequentially just slows everything down here

Reply:
So what's the conclusion here?

Reply:
@polomani Are you actually sure it will mess up the order? `Promise.all` will be consistent?

Reply:
`Promise.all` preserves the order

Reply:
@eirikdahlen Yes, this should also be `false`